aten/src/ATen/functorch/LegacyBatchingRegistrations.cpp

```diff
@@ -725,6 +725,7 @@ TORCH_LIBRARY_IMPL(aten, FuncTorchBatched, m) {
   // still legacy b/c teturns multiple tensors
   m.impl("split.Tensor", split_batching_rule);
   m.impl("split_with_sizes", split_with_sizes_batching_rule);
+  m.impl("split_with_sizes_copy", split_with_sizes_batching_rule);
```

Comment:
That doesn't sound right? One is a view while the other is not?

Reply:
Good catch. I was doing trial and error and forgot to revisit this.